### PR TITLE
Pull Request Template added

### PR DESCRIPTION
**Pull request by edgarglezc**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+_Provide link to design doc..._
+
+### Details
+
+_Add more context to describe the changes_
+
+### Checks
+
+- [ ] Did build pass?
+- [ ] Did test pass?
```
Comment (Crisspv) [inline .github/pull_request_template.md:14]: Is this change necessary?

Comment (edgarglezc) [reply .github/pull_request_template.md:14]: Im not sure

Comment (Crisspv) [reply .github/pull_request_template.md:14]: Approved

**Review by Crisspv — COMMENTED** [review]
This is a comment

**Review by Crisspv — APPROVED** [review]
